experimental/plugins/lando-lagoon/services/lagoon-redis/builder.js

```diff
@@ -10,14 +10,29 @@ module.exports = {
     version: 'custom',
     confSrc: __dirname,
     command: '/sbin/tini -- /lagoon/entrypoints.sh redis-server /etc/redis/redis.conf',
+    port: '6379',
+    portforward: true,
+    moreHttpPorts: ['6379'],
```

Comment:
@tobybellwood `moreHttpPorts` is used to tell Lando to route a `localhost:SOMEPORT` address on your host to a http interface on port `6379` on the service. Since redis does not have an http interface (that im aware of anyway) this is likely not going to work as expected. 

The `solr` service makes good use of this to provide a `localhost` address to the solr admin web ui.

Reply:
fixed up - was using it to report the forwarded port for Redis to use in local testing - but can get that from `lando info` without the URL confusion.

Reply:
yup exactly!